python/tk_desktop/desktop_window.py

```diff
@@ -17,6 +17,7 @@
 import cPickle as pickle
 import pprint
 import itertools
+import urlparse
 from collections import OrderedDict
```

Comment:
OrderedDict is 2.7+, isn't it? I guess since Desktop is bundled with 2.7 this is fine. Just mentioning it in case there's ever the possibility we won't be in 2.7.